src/System.Windows.Forms/src/System/Windows/Forms/ICommandPropertyProvider.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Windows.Input;
+
+namespace System.Windows.Forms
+{
+    internal interface ICommandPropertyProvider
+    {
+        public class CommandProviderManager
```

Comment:
If we don't expect this to be inherited let's seal it, as it'd allow the compiler to do a better job.
```suggestion
        public sealed class CommandProviderManager
```

Reply:
Outdated.